CMakeLists.txt

```diff
@@ -89,6 +88,9 @@ install(
 #############
 
 if(CATKIN_ENABLE_TESTING)
+  find_package(catkin REQUIRED COMPONENTS
+    moveit_ros_planning
+  )
```

Comment:
Don't run `find_package(catkin ..)` twice. It does not work as you think it does.

For tests, it's common practice to just call `find_package(moveit_ros_planning REQUIRED)` directly.

Then you could use `target_include_directories(..)` and `target_link_libraries(..)` on the test binaries.

Reply:
Thanks for the tip!

I added it to `include_directories` instead of `target_include_directories` because it is relevant for all (test) targets. Adding it to include directories was technically not necessary as the ros include path is already present, but it seemed good practice to add it...